src/api/dutchx.ts

```diff
@@ -86,7 +86,7 @@ async function init(): Promise<DutchExchange> {
     { sell: { address: t1 }, buy: { address: t2 } }: TokenPair,
     index: Index,
     userAccount: Account,
-  ) => dx.claimSellerFunds(t1, t2, userAccount, index, { from: userAccount })
+  ) => dx.claimSellerFunds(t1, t2, userAccount, index, { from: userAccount, gas: 4712388 })
```

Comment:
how did you get this number? `web3.estimateGas`?

Reply:
You got this number, long time ago, for similar transactions:)

Reply:
should probably check it via `estimateGas` and then `2x` it to be safe